code/game/objects/items/rcd/RCD.dm

```diff
@@ -221,6 +221,7 @@
 	current_active_effects += 1
 	_rcd_create_effect(target, user, delay, rcd_results)
 	current_active_effects -= 1
+	log_tool("used RCD with design mode: \"[rcd_results["[RCD_DESIGN_MODE]"]]\" and design path: \"[rcd_results["[RCD_DESIGN_PATH]"]]\" with delay [delay]", user)
```

Comment:
Makes sense to log only if the effect was successful.

Luckily _rcd_create_effect() proc returns TRUE if it was so you can write an if condition there and log only if that proc returns TRUE.